Program.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+
+namespace task_DEV_1
+{
+    // Class that performs the function of a counter.
+    public static class Counter

```

Comment:
I made the class static, because I do not need to create objects of class Counter.
I don't think that I 'll create classes, inherited from counter. So why I can not use static?